plugins/kubernetes/test/models/kubernetes/api/horizontal_pod_autoscaler_test.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+require_relative '../../../test_helper'
+
+SingleCov.covered!
+
+describe Kubernetes::Api::HorizontalPodAutoscaler do
+  let(:start_time) { "2017-03-31T22:56:20Z" }
+  let(:event) { {metadata: {creationTimestamp: start_time}, type: 'Normal'} }
+  let(:events) { [event] }
```

Comment:
events/event/start_time can be moved into `#events_indicating_failure ` test ... and can also remove the before and have the `event` be a warning instead